kolena/_experimental/object_detection/workflow.py

```diff
@@ -87,7 +87,7 @@ class TestSampleMetrics(MetricsTestSample):
 @dataclass(frozen=True)
 class ClassMetricsPerTestCase(MetricsTestCase):
     Class: str
-    TestSamples: int
+    n_test_samples: int
```

Comment:
We should follow the same naming convention across all metrics. What would the ideal name for this field be? `nImages`?

Reply:
I think ideal naming is `# Images` to follow the test cases, but maybe that should be renamed to `Images`, like `Objects` and `Inferences`. I think `Images` > `nImages`.